patches/tModLoader/Terraria/Main.cs.patch

```diff
@@ -2648,6 +2648,143 @@
  						if ((!expertMode || type != 266) && ((type != 439 && type != 440) || npc[num2].ai[0] != 5f)) {
  							if (type >= 134 && type <= 136) {
  								scale = 1.5f;
+@@ -34675,6 +_,7 @@
+ 			if (!CanShowInfoAccs)
+ 				return;
+ 
++			/*
+ 			bool flag = false;
+ 			bool flag2 = false;
+ 			bool flag3 = false;
+@@ -34687,6 +_,7 @@
+ 			bool flag10 = false;
+ 			bool flag11 = false;
+ 			bool flag12 = false;
++			*/
+ 			int num = -1;
+ 			int num2 = -10;
+ 			int num3 = 0;
+@@ -34698,9 +_,11 @@
+ 				num4 += 50f;
+ 			}
+ 
+-			for (int i = 0; i < 12; i++) {
++			for (int i = 0; i < InfoDisplayLoader.InfoDisplayCount; i++) {
+ 				string text2 = "";
+ 				string text3 = "";
++
++				/*
```

Comment:
Are the multi-line comments necessary?

Reply:
yes, in order to prevent the vanilla info displays from running twice

Reply:
Oh, wait. I commented on the wrong multilines.